sig-auth/README.md

```diff
@@ -37,6 +37,39 @@ The Chairs of the SIG run operations and processes governing the SIG.
 * [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-auth)
 * [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2Fauth)
 
+## Subprojects
+
+The following subprojects are owned by sig-auth:
+- **audit-logging**
+  - Owners:
+    - Tim Allclair
+- **authenticators**
+  - Owners:
+    - Mike Danese
+    - Jordan Liggitt
+- **authorizers**
+  - Owners:
+    - Jordan Liggitt
+    - Mike Danese
+    - David Eads
+- **cross-cutting-documentation**
+  - Owners:
+    - NEEDS OWNER
+- **encryption-at-rest**
+  - Owners:
+    - NEEDS OWNER
+- **node-identity-and-isolation**
+  - Owners:
+    - Mike Danese
+    - Jordan Liggitt
+- **policy-management**
+  - Owners:
+    - Tim Allclair
+    - Jordan Liggitt
+- **security-conformance**
```

Comment:
I thought we decided to cut this as a subproject? I think it makes sense for our sig to invest in this area, but I think that should just be a cross-cutting effort where each subproject works on adding relevant conformance tests.